demos/continuous_batching/README.md

```diff
@@ -16,7 +16,7 @@ make release_image GPU=1
 It will create an image called `openvino/model_server:latest`.
 > **Note:** This operation might take 40min or more depending on your build host.
 > **Note:** `GPU` parameter in image build command is needed to include dependencies for GPU device.
-> **Note:** The public image from the last release might be not compatible with models exported using the the latest export script. Check the [demo version from the last release](https://github.com/openvinotoolkit/model_server/tree/releases/2024/4/demos/continuous_batching) to use the public docker image.
+> **Note:** The public image from the last release might be not compatible with models exported using the the latest export script. Check the [demo version from the last release](https://github.com/openvinotoolkit/model_server/tree/releases/2025/0/demos/continuous_batching) to use the public docker image.
```

Comment:
We don't have 2025/0 branch yet

Reply:
I have reverted the `2024 -> 2025` changes.